open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def filter_by_year_and_month(df, year, month):
</pre><p>You implementation does not the same as the function title. You are doing <code>[inifected]</code> and <code>plot</code> as well. Please move the plot out of this function. And I think you can actually rename this as <code>get_non_infected_by_year_and_month()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>DONE</p>